tests/unit_tests/test_camera_efficiency.py

```diff
@@ -14,9 +14,10 @@
 
 
 @pytest.fixture
-def camera_efficiency_lst(telescope_model_lst, simtel_path):
+def camera_efficiency_lst(telescope_model_lst, site_model_north, simtel_path):
     camera_efficiency_lst = CameraEfficiency(
         telescope_model=telescope_model_lst,
```

Comment:
Better to keep the order of arguments here and following tests as defnied in CameraEfficiency (`label` <--> `simtel_source_path`)

Reply:
Changed it, but this shouldn't really matter. Do you think for readability?